src/components/content/HeroImage.js

```diff
@@ -0,0 +1,26 @@
+/**
+ * Decorative images
+ */
+import React from 'react'
+import givethHeroSide from '../../images/decorator-leaf.svg'
+// import co2kenHeroSide from ''
+import givethHeroMain from '../../images/people-header.svg'
+import co2kenHeroMain from '../../images/tree-planting.jpg'
```

Comment:
The more I look at the code, the less I am convinced this is a good strategy. Is the plan to have ALL white labeling efforts in one application? What if there are ten projects whitelabeling. What if one org wants to customize more? Will we keep adding switches for every little asset?

In the least I wish we would discuss things among the team instead of just taking decisions like this.